### PR TITLE
[tests/conftest.py] Add parameter for collecting dump: "since yesterday"

### PR DESCRIPTION
**Pull request by stephenxs**
### Description of PR

Summary:
The sonic dump will be collected when a test case fails.
However, it can be very time-consuming to collect dump in a switch with a lot of log files.
To collect dump since yesterday prevents DUT from consuming too much time on it.

Signed-off-by: Stephen Sun <stephens@mellanox.com>

### Type of change

- [ ] Bug fix
- [x] Testbed and Framework(new/improvement)
- [ ] Test case(new/improvement)

#### What is the motivation for this PR?
To reduce the time consumed by collecting sonic dump when a test case fails

#### How did you do it?
Add "s yesterday"

### CODE REVIEW

File: tests/conftest.py [hunk 1]
```diff
@@ -407,7 +407,7 @@ def collect_techsupport(request, duthost):
     # "function" scope
     testname = request.node.name
     if request.config.getoption("--collect_techsupport") and request.node.rep_call.failed:
-        res = duthost.shell("generate_dump")
+        res = duthost.shell("generate_dump -s yesterday")
```
Comment (wangxin) [inline tests/conftest.py:410]: Can you make the time range configurable? When config option is not supplied, use default `-s yesterday`?

Comment (stephenxs) [reply tests/conftest.py:410]: Looks lit to make the time range configurable doesn't yield too much benefit.

 - So far we don't have any test case which requires log information for more than 1 day. In this sense, this modification doesn't affect test cases current running.
 - to make the time range shorter doesn't reduce the size of tarball too much.

How do you think?

Comment (wangxin) [reply tests/conftest.py:410]: Sounds OK if there is no much improvement in dump tarball size if the time range is reduced to hours.

**Comment by wangxin** [comment]
retest this please

**Comment by stephenxs** [comment]
retest this please

**Pull request merged**